pkg/controller/provider/container/ovirt/watch.go

```diff
@@ -615,3 +615,42 @@ func (r *DiskProfileHandler) validate(profile *model.DiskProfile) {
 		return
 	}
 }
+
+// Watch for ServerCpu changes and validate as needed.
+type ServerCpuEventHandler struct {
+	libmodel.StockEventHandler
+	// DB.
+	DB libmodel.DB
+	// Logger.
+	log logging.LevelLogger
+}
+
+// ServerCpu updated.
+func (r *ServerCpuEventHandler) Updated(event libmodel.Event) {
+	serverCpu, cast := event.Model.(*model.ServerCpu)
+	if cast {
+		r.validate(serverCpu)
+	}
+}
+
+// Report errors.
+func (r *ServerCpuEventHandler) Error(err error) {
+	r.log.Error(liberr.Wrap(err), err.Error())
+}
+
+func (r *ServerCpuEventHandler) validate(serverCpu *model.ServerCpu) {
+	list := []model.Host{}
```

Comment:
Host?..

Reply:
we don't watch this endpoint, so let's drop the event handler.